src/test/ui/print_type_sizes/nullable.stdout

```diff
@@ -0,0 +1,24 @@
+print-type-size type: `IndirectNonZero<u32>` overall bytes: 40 align: 8
+print-type-size    field .pre bytes: 8
+print-type-size    field .nested bytes: 24
+print-type-size    field .post bytes: 8
+print-type-size type: `MyOption<IndirectNonZero<u32>>` overall bytes: 40 align: 8
+print-type-size    variant Some exact bytes: 40
+print-type-size        field .0 bytes: 40
```

Comment:
Sounds fine. I thought about suggesting backticks for the field names too, but the '.' seemed like a sufficient marker. But backticks around types *and* fields *and* variants is another possibility. I'll let you decide; there's not going to be One True Way to print this data :)